src/components/card/CommentsTabSidebar.vue

```diff
@@ -0,0 +1,320 @@
+						items: () => {
+							return this.currentBoard.users
+						},
+						// is called when a suggestion starts
+						onEnter: ({
+							items, query, range, command, virtualNode
+						}) => {
+							this.query = query
+							this.filteredUsers = items
+							this.suggestionRange = range
+							this.renderPopup(virtualNode)
+							// we save the command for inserting a selected mention
+							// this allows us to call it inside of our custom popup
+							// via keyboard navigation and on click
+							this.insertMention = command
+						},
+						// is called when a suggestion has changed
+						onChange: ({
+							items, query, range, virtualNode
+						}) => {
+							this.query = query
+							this.filteredUsers = items
+							this.suggestionRange = range
+							this.navigatedUserIndex = 0
+							this.renderPopup(virtualNode)
+						},
+						// is called when a suggestion is cancelled
+						onExit: () => {
+							// reset all saved values
+							this.query = null
+							this.filteredUsers = []
+							this.suggestionRange = null
+							this.navigatedUserIndex = 0
+							this.destroyPopup()
+						},
+						// is called on every keyDown event while a suggestion is active
+						onKeyDown: ({ event }) => {
+							// pressing up arrow
+							if (event.keyCode === 38) {
+								this.upHandler()
+								return true
+							}
+							// pressing down arrow
+							if (event.keyCode === 40) {
+								this.downHandler()
+								return true
+							}
+							// pressing enter
+							if (event.keyCode === 13) {
+								this.enterHandler()
+								return true
+							}
+							return false
+						},
+						// is called when a suggestion has changed
+						// this function is optional because there is basic filtering built-in
+						// you can overwrite it if you prefer your own filtering
+						// in this example we use fuse.js with support for fuzzy search
+						onFilter: (items, query) => {
+							if (!query) {
+								return items
+							}
+							const fuse = new Fuse(items, {
+								threshold: 0.2,
+								keys: ['uid', 'displayname']
+							})
+							return fuse.search(query)
+						}
+					})
+				],
+				content: '',
+				onUpdate: ({ getHTML }) => {
+					this.newComment = getHTML().replace(/(<p>|<\/p>)/g, '')
+					// this.newComment = getHTML()
+				}
+			}),
+			query: null,
+			suggestionRange: null,
+			filteredUsers: [],
+			navigatedUserIndex: 0,
+			insertMention: () => {},
+			observer: null
+
+		}
+	},
+	computed: {
+		...mapState({
+			comments: state => state.comment.comments,
+			currentBoard: state => state.currentBoard
+		}),
+
+		hasResults() {
+			return this.filteredUsers.length
+		},
+		showSuggestions() {
+			return this.query || this.hasResults
+		}
+
+	},
+	watch: {
+		'card': {
+			immediate: true,
+			handler() {
+				this.loadComments()
+			}
+		}
+	},
+	created() {
+	},
+
+	methods: {
+		loadComments() {
+			this.isLoading = true
+			this.card.limit = this.limit
```

Comment:
I'm not a fan of this approach, the limit should be moved in the store itself. The offset should also be kept in the store, so the component doesn't need to care about it. Also we need some logic to handle updating for newer comments